diff --git a/chrome/app/theme/chromium/BRANDING b/chrome/app/theme/chromium/BRANDING
--- a/chrome/app/theme/chromium/BRANDING
+++ b/chrome/app/theme/chromium/BRANDING
@@ -1,8 +1,8 @@
-COMPANY_FULLNAME=The Chromium Authors
-COMPANY_SHORTNAME=The Chromium Authors
-PRODUCT_FULLNAME=Chromium
-PRODUCT_SHORTNAME=Chromium
-PRODUCT_INSTALLER_FULLNAME=Chromium Installer
-PRODUCT_INSTALLER_SHORTNAME=Chromium Installer
-COPYRIGHT=Copyright 2017 The Chromium Authors. All rights reserved.
-MAC_BUNDLE_ID=org.chromium.Chromium
+COMPANY_FULLNAME=Sunshine OS contributors
+COMPANY_SHORTNAME=Sunshine OS
+PRODUCT_FULLNAME=Sunshine OS
+PRODUCT_SHORTNAME=Sunshine
+PRODUCT_INSTALLER_FULLNAME=Sunshine OS Installer
+PRODUCT_INSTALLER_SHORTNAME=Sunshine Installer
+COPYRIGHT=Copyright 2026 Sunshine OS contributors.
+MAC_BUNDLE_ID=com.sunshine.browser
 MAC_CREATOR_CODE=Cr24
